### ISSUE TITLE
Uninitialized constant CfnDsl::VERSION (NameError)

### ISSUE THREAD

**Issue by cmaxwellau** [stevenjack/cfndsl#246]
/usr/local/lib/ruby/gems/2.3.0/gems/cfndsl-0.11.1/bin/cfndsl:10:in `block in <top (required)>': uninitialized constant CfnDsl::VERSION (NameError)

**Comment by cmaxwellau** [comment]
Last working version was 53363b0

**Comment by kornypoet** [comment]
Looking into this.

**Comment by kornypoet** [comment]
Interesting the situation that creates this. Installing cfndsl globally and using it is fine:
```
demosthenes :: ~ % gem install cfndsl
demosthenes :: ~ % cfndsl -h
Usage: cfndsl [options] FILE
...
```

Including it in another project and invoking it is also fine:

```
demosthenes :: ~/foo % bundle install
Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Resolving dependencies...
Using cfndsl 0.11.1
Using bundler 1.12.5
Bundle complete! 1 Gemfile dependency, 2 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
demosthenes :: ~/foo % bundle exec cfndsl -h
Usage: cfndsl [options] FILE
    -o, --output FILE                Write output to file
    -y, --yaml FILE                  Import yaml file as local variables
    -r, --ruby FILE                  Evaluate ruby file before template
    -j, --json FILE                  Import json file as local variables
    -p, --pretty                     Pretty-format output JSON
    -D, --define "VARIABLE=VALUE"    Directly set local VARIABLE as VALUE
    -v, --verbose                    Turn on verbose ouptut
    -b, --disable-binding            Disable binding configuration
    -h, --help                       Display this screen
```

The only way I can reproduce the error is by invoking cfndsl from the repo _without_ using bundler:

```
demosthenes :: ~/dev/cfndsl % bin/cfndsl -h
bin/cfndsl:10:in `block in <main>': uninitialized constant CfnDsl::VERSION (NameError)
	from /Users/trdempsey/.rbenv/versions/2.3.0/lib/ruby/2.3.0/optparse.rb:1061:in `initialize'
	from bin/cfndsl:9:in `new'
	from bin/cfndsl:9:in `<main>'
```

So, I would first recommend to _not_ use cfndsl this way, but always use `bundle exec`, no matter where you are invoking. That being said, it does cover up the issue that `cfndsl/version.rb` is not being required properly. Should be a simple fix.

**Comment by gergnz** [comment]
Resolved in #247
